includes/sql-schema/update.php

```diff
@@ -153,6 +153,9 @@
         $db_rev = $filename;
         if ($insert) {
             dbInsert(array('version' => $db_rev), 'dbSchema');
+            if ($db_rev >= 6) {
+                $insert = 0;
```

Comment:
Yeah, but this code doesn't do that to my eye, it just sets $insert = 0, which is not used anywhere.